src/molgenis/capice_resources/core/errors.py

```diff
@@ -0,0 +1,6 @@
+class SampleMismatchError(Exception):
```

Comment:
The definition of IOError is very vague, as it contains none: https://docs.python.org/3/library/exceptions.html#IOError . I will change it to SampleSizeMismatchError, because I do not think that IOError is really fitting nor descriptive enough.